resources/metabase-plugin.yaml

```diff
@@ -36,6 +36,8 @@ driver:
     - name: schema-filters
       type: schema-filters
       display-name: Schemas
+    - advanced-options-start
+    - default-advanced-options
```

Comment:
Omg, that ended up being so simple in the end.

Reply:
Haha yea, but it was also a good reminder to get some other pending things updated as well!